ExotischNederland/Models/Answer.cs

```diff
@@ -1,13 +1,43 @@
-﻿using System;
+﻿using ExotischNederland.DAL;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace ExotischNederland.Models
 {
-    internal class Answer
+    public class Answer
     {
-        private string tablename = "Answer";
+        public int Id { get; set; }
+        public Question Question { get; set; }
+        public string Text { get; set; }
+        public bool Correct { get; set; }
+
+        private Answer(int id, Question question, string text, bool correct)
+        {
+            Id = id;
+            Question = question;
+            Text = text;
+            Correct = correct;
+        }
+
+        // Static factory method to create a new Answer and save it to the database
+        public static Answer CreateAnswer(Question question, string text, bool correct)
+        {
+            Answer newAnswer = new Answer(0, question, text, correct);
+
+            // Save to database
+            SQLDAL db = new SQLDAL();
+            var values = new Dictionary<string, object>
+        {
```

Comment:
Careful with the line indentation here. The { and } should be 1 tab further for readability